cpp/modmesh/buffer/SimpleArray.hpp

```diff
@@ -729,6 +753,68 @@ class SimpleArray
     value_type * m_body = nullptr;
 }; /* end class SimpleArray */
 
+template <typename A, typename T>
```

Comment:
Please make the function implementation/definition adjacent to the class declaration of the mixin.  This new mixin is the only one that has function definition outside the class declaration.

Reply:
I see, you are talking about [line 781](https://github.com/solvcon/modmesh/pull/456/files#diff-bf43ff74a749046af0d1aca3542c01ba73e98d3065d0cabb4fc3f6ebc1f85f7dR781).

What about keeping only that function after `SimpleArray`?

No, we are not ready to make the template structure so complex.  We may consider that in the future, but not now.